parsers/postgresql/postgresql_test.go

```diff
@@ -0,0 +1,150 @@
+package postgresql
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/honeycombio/honeytail/event"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSingleQueryParsing(t *testing.T) {
+	testcases := []struct {
+		in           string
```

Comment:
could you add a `description` field here that describes what the test is looking for? then in the testing below, include that field in the output when a test fails.